app.js

```diff
@@ -124,6 +126,8 @@ router.route(app, fact);
 
 var apiDocsHtml = marked(fs.readFileSync("README.md", "utf8"));
 
+app.use("/js", express.static(__dirname + "/node_modules/jquery-mousewheel"));
```

Comment:
Hey @paigegodfrey - can you explain to me why this doesn't break the links to other front-end JS files? e.g. the link to the jquery counter? I'm assuming everything works fine, i just don't quite understand why.

Reply:
So from what I found online, because the path 'public/js/jquery.mousewheel.js' does not exist (unlike the other '/js' paths which _are_ housed within vendor assets), this line gets treated as creating a new virtual path prefix. The middleware gets triggered in response to the initial 404 error for the file not existing at 'js/jquery.mousewheel.js' as originally specified in the index.html file.

source: https://expressjs.com/en/starter/static-files.html

Reply:
ok great thank you for this explanation!